test/e2e/app-dir/ppr-full/app/dynamic/force-dynamic/page.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Suspense } from 'react'
+import { Dynamic } from '../../../components/dynamic'
+
+export const dynamic = 'force-dynamic'
+export const revalidate = 60
```

Comment:
What is this checking for? Doesn't the `force-dynamic` cancel out our `revalidate: 60`?